src/components/Organisms/TodoCard/index.jsx

```diff
@@ -1,9 +1,22 @@
-import React, { useState } from "react";
+import React, { useEffect, useState } from "react";
 import styled from "styled-components";
 import Task from "../../Molecules/Task/index.jsx";
 import AddTaskButton from "../../Atoms/AddTaskButton/index.jsx";
 const TodoCard = () => {
   const [taskList, setTaskList] = useState([]);
+  useEffect(() => {
+    const getItemTaskList = localStorage.getItem("taskList");
```

Comment:
```suggestion
    const taskList = localStorage.getItem("taskList");
```
変数名は中身を適切に表現したものが好ましいです。
`getItemTaskList` だと、タスクリストを取得する関数かのような印象を与えると思います。
自分だったら上のようにつけるかなと思います。

Reply:
ただ、上のスコープでも `taskList` が宣言されている（6行目）ので、それと変数名が被るのを避けるために `loadedTaskList` とかでもいいかもしれません。